Src/LibHoney/Transmission.cs

```diff
@@ -89,7 +111,14 @@ public void Send (Event ev)
                 success = pending.TryAdd (ev);
```

Comment:
does it make sense to clone the event object to keep it from being modified on whatever thread did the `Send`ing? after passing it off to the `Send()` method?